File: tests/test_data/ChatGPT/v1/Transcribed_JS/Suma.js
const readline = require('readline').createInterface({
  input: process.stdin,
  output: process.stdout
});

class Suma {
  constructor() {
    this.obtenerNumeros();
  }

  async obtenerNumeros() {
    const num1 = await this.leerNumero('Ingrese el primer número: ');
    const num2 = await this.leerNumero('Ingrese el segundo número: ');
    const suma = num1 + num2;
    console.log(`La suma es: ${suma}`);
    readline.close();
  }

  leerNumero(mensaje) {
    return new Promise((resolve) => {
      readline.question(mensaje, (num) => {
        resolve(parseInt(num));
      });
    });
  }
}

new Suma();

File: tests/test_data/ChatGPT/v1/Transcribed_PY/Suma.py
def suma_numeros():
    try:
        num1 = int(input("Ingrese el primer número: "))
        num2 = int(input("Ingrese el segundo número: "))
        suma = num1 + num2
        print(f"La suma es: {suma}")
    except ValueError:
        print("Error: Por favor ingrese un número válido")

suma_numeros()

if __name__ == "__main__":
    suma_numeros()